lib/request-log.js

```diff
@@ -0,0 +1,59 @@
+var format = require('util').format;
```

Comment:
@thisandagain we'd still have to write a serializer for hapi request objects, wouldn't we?